crates/ruff_python_parser/scripts/fuzz_parser.py

```diff
@@ -0,0 +1,214 @@
+"""
+Run the parser on randomly generated (but syntactically valid) Python source-code files.
+
+The following dependencies must be installed into a Python environment to run the script:
+- `pysource-codegen`
+- `pysource-minimize`
+- `termcolor`
+- A released version of `ruff` itself (only necessary if you pass the `--only-new-bugs` flag)
```

Comment:
As mentioned, we can include a `requirements.in` (and `uv` generated `requirements.txt`) file for this)